internal/backend/service/interceptor/auth.go

```diff
@@ -3,36 +3,58 @@ package interceptor
 import (
 	"context"
 	"fmt"
+	"strings"
 
+	"connectrpc.com/connect"
+	"github.com/golang-jwt/jwt/v5"
 	snitchv1 "snitch/pkg/proto/gen/snitch/v1"
 	"snitch/pkg/proto/gen/snitch/v1/snitchv1connect"
-
-	"connectrpc.com/connect"
 )
 
 type contextKey string
 
 const (
-	ServerIDHeader     = "X-Server-ID"
 	serverIDContextKey = contextKey("server_id")
 	groupIDContextKey  = contextKey("group_id")
 )
 
-func getServerID(req connect.AnyRequest) (string, error) {
```

Comment:
If we remove the server ID header, will we then place it in the arguments for every event? I'm fine with this, I jsut haven't seen this done yet